user_docs/en/userGuide.t2t

```diff
@@ -1220,6 +1220,16 @@ By default, NVDA will detect the formatting at the position of the System caret
 
 Enable this option while proof reading documents in applications such as Microsoft Word, where formatting is important.
 
+==== Report line indents With ====
+the Report line indents with combo box has four options.
+
+- Off: NVDA will not treat indents in text specially.

```

Comment:
Again, be careful with the distinction between line indentation and paragraph indentation here. Because this is within the line indentation section, you can probably just change "indents in text" to "indentation".